src/Dropdown/Dropdown.tsx

```diff
@@ -361,121 +361,119 @@ const Dropdown: React.FunctionComponent<DropdownProps> = (props: DropdownProps):
     }
 
     return (
-        <>
-            <div className={`dropdown custom-dropdown${shouldDisable ? " disabled" : ""}${props.className ? " " + props.className : ""}`} id={props.id}>
-                {props.label && <label className="dropdown-label">{props.label}</label>}
-
-                <div
-                    onKeyDown={shouldDisable ? null : handleKeyDownToggle}
-                    ref={dropdownToggleRef}
-                    className={`btn btn-secondary custom-dropdown-toggle${open ? " open" : ""}${props.more ? " more mx-right" : ""}${shouldDisable ? " disabled" : ""}`}
-                    id={id}
-                    aria-label={`Dropdown toggle: ${getTitleLabel()}`}
-                    aria-haspopup={true}
-                    aria-expanded={open}
-                    tabIndex={shouldDisable ? -1 : 0}
-                    onClick={shouldDisable ? null : handleClickToggle}
-                >
-                    {!props.more ? (
-                        <>
-                            <div className="title">{getTitleLabel()}</div>
-
-                            <div className="right-items">
-                                {(props.clearable || props.multi) && selectedList.length > 0 ? (
-                                    <div className="dropdown-icon-holder" onClick={shouldDisable ? null : handleClickClear}>
-                                        {timesIcon}
-                                    </div>
-                                ) : null}
-                                <div className="dropdown-icon-holder chevron">{chevronDownIcon}</div>
-                            </div>
-                        </>
-                    ) : (
+        <div className={`dropdown custom-dropdown${shouldDisable ? " disabled" : ""}${props.error ? " has-error" : ""}${props.className ? " " + props.className : ""}`} id={props.id}>
```

Comment:
is there a way to use classNames dep here ?

Reply:
I'm only using it in the refactoring branch, this is branched out of master as a **hotfix**